### PR TITLE
{bio}[foss/2022b] leafcutter v0.2.9 w/ R 4.2.2 + additional required extensions in R-bundle-Bioconductor v3.16

### PR DESCRIPTION
**Pull request by laraPPr**
(created using `eb --new-pr`)

requires:
* #18568

### CODE REVIEW

**Comment by laraPPr** [comment]
Test report by @laraPPr
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in total)
node3541.doduo.os - Linux RHEL 8.6 (Ootpa), x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.10.8
See https://gist.github.com/laraPPr/0dff3d81c269b35c205a57954d0820f2 for a full test report.

File: easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.16-foss-2022b-R-4.2.2.eb [hunk 1]
```diff
@@ -1241,12 +1241,19 @@ exts_list = [
     ('ANCOMBC', '2.0.2', {
         'checksums': ['e577ffe62a86090de9ea3743c84a4f3b34c3d645dcedff3c8af675b8c9b69136'],
     }),
-    ('decoupleR', '2.4.0', {
-        'checksums': ['b141afd331449b0b1c2dc3dc3097465c2c02b5026129d8f06dd5bfb445bce2a0'],
+    ('intervals', '0.15.4', {
+        'checksums': ['50c0e1e3aab3e7b72cc1f0a6559d96caa3a360e969c38538479907e6cbe39f8f'],
     }),
-    ('UCell', '2.2.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.16-foss-2022b-R-4.2.2.eb:1247]: You're (probably accidentally) removing extensions here (probably you were using an older copy of this easyconfig, see #18552), please restore `UCell` and `decoupleR`.

Ideally adding extensions to `R-bundle-Bioconductor` is done in a separate PR so it stands out a bit, but we'll go ahead as is here.

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--skip"

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--skip"

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--sanity-check-only"

**Comment by boegel** [comment]
`--sanity-check-only` failed with "`/bin/bash: R: command not found`" because of a bug in `RPackage` easyblock, which makes it incompatible with `--sanity-check-only`, similar to what we had for `PythonPackage` before https://github.com/easybuilders/easybuild-easyblocks/pull/2828 was fixed...

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--skip"

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="leafcutter-0.2.9-foss-2022b-R-4.2.2.eb"

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @laraPPr!

**Pull request merged**